Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+def fail(reason) {
+  def pr_branch = ''
+  if (env.CHANGE_BRANCH != null) {
+    pr_branch = " (${env.CHANGE_BRANCH})"
+  }
+  error("${reason}")
+}
+
+node('lisk-explorer-01'){
+  try {
+    stage ('Prepare Workspace') {
+      deleteDir()
+      checkout scm
+    }
+
+    stage ('Shellcheck') {
+      try {
+        sh '''#!/bin/bash
+        # shellcheck
+        shopt -s globstar; /opt/shellcheck/shellcheck **/*.sh
+        '''
+      } catch (err) {
+        echo "Error: ${err}"
+        fail('Stopping build, installation failed')
+      }
+    }
+  } catch(err) {
+    echo "Error: ${err}"
+  } finally {
```

Comment:
Looks like the whole `finally` block does nothing excpet for the `deleteDir()` call on success. Do we really need it given the size of this workspace?